### ISSUE TITLE
Results Scroll Too Quickly

### ISSUE THREAD

**Issue by eytanweber** [Wox-launcher/Wox#2863]
Hello, I want to say I love this program and I've been looking for something like it for a while. There are just two things I was wondering about.

First, is there any way to change how quickly results scroll? When I search something in the main window, it shows results, but even the slightest scroll on my trackpad flies all the way down to the bottom of the list. It's just very sensitive. Is there a way to fix that?

Second, is there a way to search, and then narrow it down by file type? One of the few things I miss from the regular Windows 10 search is that it split results into categories. So, if I was looking for a folder, but it wasn't one of the ones in the short list, I could click on "Folders" so I would only see folder results. Is that possible?

![Annotation 2020-04-24 120613](https://user-images.githubusercontent.com/47161277/80233173-12c3fc00-8624-11ea-983a-c3740ecbc7b4.jpg)

**Comment by bao-qian** [comment]
for your second question, you need install everything 
1. Download .exe installer https://www.voidtools.com/
2. make sure "Install Everything Service" is checked

then search like
folder: folder query
or
file: file query

full list file search syntax: https://www.voidtools.com/support/everything/searching/



for you first feature request, you need to open it in a separate issue and give more info.
1. a video to show the scroll speed
2. wox version
3. your hardware version, tackpad version, driver etc.

**Comment by eytanweber** [comment]
I apologize, but I don't think this has been resolved yet. I already have Everything running (search works fine). I looked through all the settings in Everything, and can't seem to find anything that allows me to do a search in Wox, then select an option to only show certain file types to narrow down the search from there. For example, if I search "Program" to find the "Program Files" folder, it'll bring up a lot of options of different file types. What I want to do is then tell Wox to only show me folders that have "Program" in them. I can't find a way to do do that with either Wox or Everything.
